eq-author-api/schema/typeDefs.js

```diff
@@ -97,6 +100,8 @@ type QuestionPage implements Page {
   confirmation: QuestionConfirmation
   routing: Routing2
   totalValidation: TotalValidationRule
+  validationErrorInfo: ValidationErrorInfo
+  isNew: Boolean
```

Comment:
Same here, the `isNew` should be a mandatory boolean.

Reply:
done